### DISCUSSION TITLE
local file change error running update_windows.bat

### DISCUSSION THREAD

**Discussion by techbutler** [oobabooga/text-generation-webui#3742] [Q&A]
I had made some edits to download-model.py which have since been deleted, but am getting the below error nonetheless. It doesn't seem to be giving me a choice to be doing anything about it. It just exits the update.

error: Your local changes to the following files would be overwritten by merge:
        download-model.py
Please commit your changes or stash them before you merge.
Aborting
Updating feecd81..9a202f7
Command '"E:\oobabooga_windows\installer_files\conda\condabin\conda.bat" activate "E:\oobabooga_windows\installer_files\env" >nul && git pull' failed with exit status code '1'. Exiting...

Done!
Press any key to continue . . .

**Comment by missionfloyd** [comment]
Run `git restore .` to undo changes.
